tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -419,11 +419,12 @@ public function testPrepareQueryBindValueDateTimeType()
 
     /**
      * @group DBAL-78
+     * @group DBAL-2685
      */
     public function testNativeArrayListSupport()
     {
         for ($i = 100; $i < 110; $i++) {
-            $this->_conn->insert('fetch_table', array('test_int' => $i, 'test_string' => 'foo' . $i, 'test_datetime' => '2010-01-01 10:10:10'));
+            $this->_conn->insert('fetch_table', array('test_int' => $i, 'test_string' => 'foo' . $i, 'test_datetime' => '2010-01-01 10:10:' . str_pad($i - 100, 2, "0", STR_PAD_LEFT)));
```

Comment:
I had to change this because the value wasn't based on $i, this is just improving the way the test is working